app/assets/javascripts/member-facing/redirectors.js

```diff
@@ -0,0 +1,43 @@
+const RegisterMemberRedirector = {
+  attemptRedirect:  function(followUpUrl, member) {
```

Comment:
you can format this as `attemptRedirect(followUpUrl, member) {` and it'll be consistent with the other objects